fusor-ember-cli/app/routes/deployment-new/satellite/configure-organization.js

```diff
@@ -11,14 +11,11 @@ export default Ember.Route.extend(DeploymentNewSatelliteRouteMixin, {
     var self = this;
     controller.set('model', model);
     controller.set('showAlertMessage', false);
-    this.store.findAll('organization').then(function(results) {
-      controller.set('organizations', results);
-      if (results.get('length') === 1) {
-        var defaultOrg = results.get('firstObject');
-        controller.set('model', defaultOrg);
-        controller.set('selectedOrganization', defaultOrg);
-        self.modelFor('deployment-new').set('organization', defaultOrg);
-      }
+    this.store.findRecord('organization', 1).then(function(result) {
```

Comment:
I thought it was better to use id =1 than the name "Default Organization" which can change without breaking the code.